Xamarin.Forms.Core/Markup/LayoutExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Xamarin.Forms.Markup
+{
+	public static class LayoutExtensions
+	{
+		public static TLayout Padding<TLayout>(this TLayout layout, Thickness padding) where TLayout : Layout
+		{ layout.Padding = padding; return layout; }
+
+		public static TLayout Padding<TLayout>(this TLayout layout, double horizontalSize, double verticalSize) where TLayout : Layout
+		{ layout.Padding = new Thickness(horizontalSize, verticalSize); return layout; }
+
+		public static TLayout Paddings<TLayout>(this TLayout layout, double left = 0, double top = 0, double right = 0, double bottom = 0) where TLayout : Layout
```

Comment:
To prevent ambiguity errors when people do not use named parameters.
E.g. all of the below padding variations are unambiguous:
```CSharp
new StackLayout { }
                 .Padding (5)
                 .Padding (5, 10)
                 .Paddings (5, 10, 8)
                 .Paddings (5, 10, 8, 12)
                 .Paddings (bottom: 12)
```

If we were to name these methods the same, the first two (quite common) variations would no longer be possible.